index.html

```diff
@@ -3,22 +3,14 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="shortcut icon" href="/favicon.ico">
     <link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Inconsolata:400,700|Open+Sans:300italic,400italic,600italic,400,300,600">
 
-    <!--
-      Notice the use of %PUBLIC_URL% in the tag above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favico.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
```

Comment:
not sure if this was for Surge or if @youknowriad would still know why it was necessary.

after the update the favicon still works as in `master` for local builds, and shouldn't change in production. surprisingly though this doesn't make it to the public production build because the `theme/console3.php` file doesn't specify a favicon 🤦‍♂️

Reply:
It does sound generated, to be honest I don't really remember much about this project 😬

Reply:
The `%PUBLIC_URL%` thing appears to be a part of create-react-app's infrastructure. https://create-react-app.dev/docs/using-the-public-folder/